File: c/one.c
#include <stdio.h>
#include <math.h>

int is_prime (int m) {
	int status = 1;
	int i = 3;
	float lim = sqrt(m);
	while ((i <= lim) && status) {
		if (m % i == 0) {
			status = 0;
		} else {
			i += 2;
		}
	}
	return status;
}

long suma_prima (int lim) {
	long suma = 10;
	int i = 7;
	while (i < lim) {
		short status = 1;
		int j = 3;
		while ((j*j <= i) && status) {
			if (i % j == 0) {
				status = 0;
			} else {
				j += 2;
			}
		}
		if (status) {
			suma += i;
		} ;
		i += 2;
	}
	return suma;
}

int main() {
	printf("%ld", suma_prima(2000000));
}

File: python/one.py
import time

def sqr (x) :
    return x * x

def is_div (a,b) :
  return 0 == a % b

def sqrt (x) : 
	return x**(0.5)

def is_prime (m) :
    status = True
    lim = sqrt (m)	
    i = 3
    while (i <= lim) and status:
        if (is_div (m,i)):
            status = False
        else :
            i = i + 2
    return status
	
def suma_prima (lim) :
    suma = 10
    i = 7
    cur = time.time ()
    while i < lim :
        if is_prime (i) :
            suma += i
        i += 2
    print time.time () - cur, "second"  
    return suma
        



		
		
		
		
		
